### PR TITLE
Add Radxa Zero to download page

### PR DESCRIPTION
**Pull request by almirus**

### CODE REVIEW

**Comment by almirus** [comment]
radxazero

image for radxa zero

File: index.html [hunk 1]
```diff
@@ -1018,6 +1019,27 @@ <h1>Download</h1>
...
+								<div><span>RAM:</span>512 MiB - 4 GiB DDR4</div>
+								<div><span>USB:</span>USB 2.0 OTG and power, USB 3.0 HOST</div>
+								<div><span>NET:</span>Onboard WiFi 4/5 (802.11 a/b/g/n/ac), Bluetooth 4/5</div>
+								<div><span>Storage:</span>8 - 128 GiB eMMC</div>
+								<br>
+								<div><a href="https://dietpi.com/downloads/images/DietPi_RadxaZero-ARMv8-Bullseye.7z"><span>Download Image:</span><i class="fa fa-download"></i><u>Download</u></a></div>
+								<div><a href="https://dietpi.com/docs/install/" target="_blank" rel="noopener"><span>How to install / Details:</span><i class="fa fa-book"></i><u>View</u></a></div>
+							</div>
+							<p>Radxa Zero features a quad core 64 ARM processor, 32bit LPDDR4, up to 4K@60 HDMI, 802.11 ac WIFI, Bluetooth 5.0, USB 3.0, 40-pin GPIO header. Radxa Zero supports USB 2.0 OTG&power via one USB C port.<br>
+								Radxa Zero comes in 4 options with different ram/storage configuration.</p>
```
Comment (MichaIng) [inline index.html:1038]: Some possible capitalisation issues below: "WiFi" and "RAM" (capital letters) is needed to pass the spell check.

File: index.html [hunk 2]
```diff
@@ -1089,6 +1111,13 @@ <h3>Pinebook Pro</h3>
 							<p>RK3399 1.8GHz | 6 cores | ARMv8</p>
 						</a>
 					</div>
+					<div class="col-xl-2 col-lg-3 col-md-4 col-6 mix radxa">
+						<a href="#downloadinfo" class="thumbnail show_hide" rel="#radxa">
+							<img src="images/sbc_images/radxazero.jpg" alt="Radxa Zero photo" width="8" height="5" loading="lazy">
+							<h3>Radxa Zero</h3>
+							<p>Amlogic S905Y2 1.8GHz | 4 cores | ARMv8</p>
+						</a>
+					</div>
```
Comment (MichaIng) [inline index.html:1120]: Would be good to move this below ROCK Pi S, so all Radxa tiles are shown together.

**Comment by MichaIng** [comment]
Many thanks! And awesome how detailed the information is that you provided. I actually wanted to wait for developer samples to arrive before publishing the image on our website, also recreating it with newest kernel. However, I'll rebuild the image now, and if you can confirm that it boots and works find, we can push this live.

Another issue is that there is no kernel meta package yet, hence kernel upgrades need to be searched and installed manually currently:
- testing suite: https://github.com/radxa/apt/tree/gh-pages/buster-testing/pool/main/l
- stable suite: https://github.com/radxa/apt/tree/gh-pages/buster-stable/pool/main/l

But as long as the latest kernel works fine, this is acceptable. We can offer to install the kernel meta package on DietPi update, once is is available.

File: index.html [hunk 3]
```diff
@@ -1018,6 +1019,27 @@ <h1>Download</h1>
...
+				<div id="radxa" class="toggleDiv single-project">
+					<div class="row"><div class="col-md-12 project-title"><h3>Radxa zero</h3><span class="show_hide close"></span></div></div>
+					<div class="row">
+						<div class="col-md-6 project-description">
+							<div class="project-info">
+								<div><span>CPU:</span>Amlogic S905Y2 1.8 GHz | 4 cores | ARMv8</div>
+								<div><span>RAM:</span>512 MiB - 4 GiB DDR4</div>
+								<div><span>USB:</span>USB 2.0 OTG and power, USB 3.0 HOST</div>
+								<div><span>NET:</span>Onboard WiFi 4/5 (802.11 a/b/g/n/ac), Bluetooth 4/5</div>
+								<div><span>Storage:</span>8 - 128 GiB eMMC</div>
```
[the remark this answers is not in the document]
Comment (almirus) [reply index.html:1032]: No 😎
https://wiki.radxa.com/Zero/hardware/models

File: index.html (hunk 2 above)
Comment (almirus) [reply index.html:1120]: maybe or not
ROCK Pi S based on Rockchip 
Radxa Zero on Amlogic
I do not know which is more correct

File: index.html (hunk 1 above)
Comment (almirus) [reply index.html:1038]: It should be "WIFI", "RAM"?

**Comment by almirus** [comment]
> However, I'll rebuild the image now, and if you can confirm that it boots and works find, we can push this live.

<html>
<body>


DietPi_RadxaZero-ARMv8-Bullseye.7z | 2021-10-26 22:13 | 115M
-- | -- | --



</body>
</html>

File: index.html (hunk 3 above)
[the remark this answers is not in the document]
Comment (MichaIng) [reply index.html:1032]: No? The link just verifies it 🤔:
SKU | RAM | eMMC | WiFi/BT | GPIO Header | Antenna | Note
-- | -- | -- | -- | -- | -- | --
RS102-D0H | 512MB | - | AP6212 | Yes | on board | 🔴
RS102-D1H | 1GB | - | AP6212 | Yes | on board | 🟠
RS102-D2E8H | 2GB | 8GB | AP6256 | Yes | on board | 🟡

- No eMMC on 512 MiB and 1 GiB model, only 8 GiB eMMC for 2 GiB RAM model, larger eMMC only available for 4 GiB RAM model.

File: index.html (hunk 1 above)
Comment (MichaIng) [reply index.html:1038]: "WiFi" and "RAM":
- https://en.wikipedia.org/wiki/Wi-Fi (we omit the dash, so basically like in the logo)
- https://en.wikipedia.org/wiki/Random-access_memory

File: index.html (hunk 2 above)
Comment (MichaIng) [reply index.html:1120]: We sort it by manufacturer, not by SoC, so this is to keep the sorting that is already present.

**Comment by MichaIng** [comment]
I'm starting with the image now. As this is done in a container and I'm in a meeting in between, give me some hours.

File: index.html (hunk 2 above)
Comment (almirus) [reply index.html:1120]: image
here?

Comment (MichaIng) [reply index.html:1120]: Yeah exactly, but also when `[All]` are shown, they should show up together, which is assured by by sorting those tile blocks accordingly.

**Comment by MichaIng** [comment]
I added the image to the base branch: https://github.com/MichaIng/DietPi-Website/commit/2cf6c02

**Comment by MichaIng** [comment]
New image is up: https://dietpi.com/downloads/images/testing/DietPi_RadxaZero-ARMv8-Bullseye.7z
I added the `fdt resize 65536` fix we found in the other issue, but I removed the `kernel_comp_addr_r` workaround, which was hopefully fixed with the latest kernel image. The device tree overlay for UART was present by default in the new image 👍.

File: index.html [hunk 4]
```diff
@@ -223,6 +223,7 @@ <h1>Download</h1>
 					<button class="nav-link filter" data-filter=".allo">Allo</button>
 					<button class="nav-link filter" data-filter=".asus">ASUS</button>
 					<button class="nav-link filter" data-filter=".nanopi">NanoPi</button>
+					<button class="nav-link filter" data-filter=".radxa">Radxa</button>
```
Comment (MichaIng) [inline index.html:226]: Ah this doubles with four lines above. The text there is "ROCK Pi" and indeed it makes sense to rename it to "Radxa".

**Comment by almirus** [comment]
> New image is up

DietPi v7.8.2 boot normal!

**Comment by MichaIng** [comment]
Let's get this published.

**Pull request merged**